source/lucidity/template.py

```diff
@@ -2,13 +2,16 @@
 # :copyright: Copyright (c) 2013 Martin Pengelly-Phillips
 # :license: See LICENSE.txt.
 
+import os
+import re
+import imp
 import abc
 import sys
-import re
+import uuid
 import functools
 from collections import defaultdict
 
-import lucidity.error
```

Comment:
I deliberately favour absolute imports these days as they keep the namespacing visible and improve clarity. 

By 'vendoring' I presume you mean including a package inside another package?